src/elements/gallery/Gallery.tsx

```diff
@@ -2,7 +2,23 @@ import React from "react";
 import PropTypes from "prop-types";
 import styles from "./Gallery.module.scss";
 
-const Gallery = ({ imageLeft, imageCenter, imageRight }) => {
+interface Props {
+  imageLeft?: string;
+  imageCenter?: string;
+  imageRight?: string;
+}
+
+const defaultProps: Props = {
```

Comment:
https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/default_props/
based on this article, we don't need to declare defaultProps since
`const Gallery: React.FC<Props> = ({
  imageLeft = "",
  imageCenter = "",
  imageRight = "",
}`

do the same